service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/Party.java

```diff
@@ -19,18 +19,18 @@
 @NoArgsConstructor(force = true)
 public class Party {
     // REFACTOR: 03/12/2019 This needs to be private, effects tests as well
```

Comment:
Think this comment can go now, seeing as you have done this restricting of access

Reply:
done